src/problems/SphericalCollapse/spherical_collapse.cpp

```diff
@@ -45,6 +44,7 @@ template <> struct Physics_Traits<CollapseProblem> {
 	// face-centred
 	static constexpr bool is_mhd_enabled = false;
 	static constexpr int nGroups = 1; // number of radiation groups
+	static constexpr UnitSystem unit_system = UnitSystem::CGS;
```

Comment:
This is not correct. `G=1` for this problem:
https://github.com/quokka-astro/quokka/blob/7a8dc754e30658c1c3c85a8173feb0baf8e692d0/tests/SphericalCollapse.in#L30

Reply:
Corrected.

Reply:
I also updated the description to note that G is no longer a runtime parameter.